src/Stryker.Core/Stryker.Core.UnitTest/Compiling/RollbackProcessTests.cs

```diff
@@ -43,7 +43,7 @@ public string Subtract(string first, string second)
             var annotatedSyntaxTree = syntaxTree.GetRoot()
```

Comment:
RollbackProcessTests should have new tests for rolling back conditional statements